src/components/Welcome/index.tsx

```diff
@@ -68,14 +69,17 @@ const WelcomeActions: FunctionComponent<WelcomeActionsProps> = ({
   )
 }
 
-const Welcome: FunctionComponent<Props> = ({
-  title,
-  descriptions,
-  nextButton,
-  nextStep,
-  translate,
-}) => {
-  const actions = <WelcomeActions {...{ nextButton, nextStep, translate }} />
+const Welcome: FunctionComponent<StepComponentBaseProps> = ({ nextStep }) => {
+  const { steps } = useSdkOptions()
+  const { translate } = useContext(LocaleContext)
+
+  const { options } = steps.find(
```

Comment:
Out of scope: I think we should introduce a distinction between global options and step options at some point. We could have a context just for step options so that each step knows about the option that belong to it. Also, I think we should rename options to `globalConfig` and `stepConfig` but this will be a breaking change in the public API

Reply:
Makes sense. The specific step options could help us avoid the long ugly `StepConfig` cherry-pick from the `steps` config!